cloud-gateway/src/main/java/com/registry/cloudgateway/dto/RestResponsePage.java

```diff
@@ -0,0 +1,37 @@
+package com.registry.cloudgateway.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RestResponsePage<T> extends PageImpl<T> {
+
+    private static final long serialVersionUID = 3248189030448292002L;
+
+    @JsonCreator(mode = JsonCreator.Mode.PROPERTIES)
+    public RestResponsePage(@JsonProperty("content") List<T> content, @JsonProperty("number") int number, @JsonProperty("size") int size,
```

Comment:
Considering that the variables from the parameter list of this method have the same names as the attributes of their corresponding @JsonProperty annotations, do we even need the annotations?